### PR TITLE
feat: add a new buildspec configuration that runs both unit and integ tests

### PR DESCRIPTION
**Pull request by shivlaks**
motivation: rather than running integ and unit tests in separate CodeBuild projects and for PR builds,
consolidate them into a single project which will run the sequentially. This is mostly to simplify the
current infrastructure setup and make PRs less confusing as we will 1 set of build logs for every PR build.

The current PR build runs these separately and posts logs/comments for each separately, which is more tedious.

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: buildspec_pr.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - pip install setuptools wheel twine
+      - python setup.py sdist
+      - pip install tox
```
Comment (wong-a) [inline buildspec_pr.yaml:8]: Can this be in the first `pip` command?

Comment (shivlaks) [reply buildspec_pr.yaml:8]: good call, i'll give that a try, but am unsure whether there's motivation behind the current structure which I reused.

Comment (shivlaks) [reply buildspec_pr.yaml]: verified that it can. updated the PR

**Comment by shivlaks** [comment]
closing in favour of using it in infrastructure code rather than adding it to the repo

**Pull request closed without merging**